bin/main.dart

```diff
@@ -123,6 +236,7 @@ Future<Map<String, String>> _loadStrings(String lang, String dirPath) async {
   return localizedStrings;
 }
 
+// Writing translated files
 _updateContent(Map<String, Map<String, String>> langStrMap, dirPath) {
```

Comment:
I don't understand what this function does. The jsonStr is not used...

Reply:
It was used in the last meaningful line. Got rid of a variable.